lib/std/zig/parser_test.zig

```diff
@@ -72,21 +72,6 @@ test "zig fmt: rewrite callconv(.Inline) to the inline keyword" {
     );
 }
 
-// TODO Remove this after zig 0.9.0 is released.
-test "zig fmt: rewrite @byteOffsetOf to @offsetOf" {
```

Comment:
The actual rewriting this is testing should also be removed.

Reply:
Ah I see, will sort that out in a sec.

Reply:
Should be ok now :)